source/MaterialXCore/Interface.cpp

```diff
@@ -305,7 +305,7 @@ InputPtr Input::getInterfaceInput() const
 {
     if (hasInterfaceName())
     {
-        ConstNodeGraphPtr graph = getAncestorOfType<NodeGraph>();
+        ConstGraphElementPtr graph = getAncestorOfType<GraphElement>();
```

Comment:
This is just an incorrect check as we want GraphElements and not just NodeGraphs.